models/dish.go

```diff
@@ -4,7 +4,6 @@ import "gorm.io/gorm"
 
 type Dish struct {
 	gorm.Model
-	Id        uint `gorm:"primaryKey;autoIncrement"`
-	Name      string
-	ImagePath *string
+	Id   uint `gorm:"primaryKey;autoIncrement"`
+	Name string
```

Comment:
why we removed imagePath?

Reply:
Cause it become on meal , as per as last schema